.github/labeler.yml

```diff
@@ -0,0 +1,30 @@
+- name: Labeler
+  uses: actions/labeler@v3
+lang/ko:
+- content/ko/*
+lang/bn:
+- content/bn/*
+lang/en:
+- content/en/*
+lang/es:
+- content/es/*
+lang/ar:
+- content/ar/*
+lang/de:
+- content/de/*
+lang/pt:
+- content/pt-br/*
```

Comment:
I also found that @seokho-son suggested 
> [lang/pt](https://github.com/cncf/glossary/labels/lang%2Fpt) : content/**pt-br**/*

in [this comment](https://github.com/cncf/glossary/issues/337#issuecomment-1038577614),

but one can see that **pt**, not **pt-br** directory exists in [`content/`](https://github.com/cncf/glossary/tree/main/content).

So I think that CNCF Glossary maintainers should choose one of these: **`content/pt`** or **`content/pt-br`** and apply. 😊

Reply:
@jihoon-seo 
The current lang directories in `main` branch are obsoleted except `en`.
Actual directory lives in a localization dev branch.
https://github.com/cncf/glossary/tree/dev-pt/content/pt-br

`pr-br` is the right one :)

That directory will be merged into main branch when PT team is ready. :)